### DISCUSSION TITLE
Image field not selected by the query (ImageColumn)

### DISCUSSION THREAD

**Discussion by rafli-sss** [rappasoft/laravel-livewire-tables#1575] [Q&A]
Output always null and when I check in laravel-debugbar "image" is not selected in the query
```php
ImageColumn::make(__('Image'), 'image')
  ->location(
      fn (Menu $menu) => dd($menu->image) //asset('storage/img/menu/' . $menu->image)
    ),

// Without type hint
ImageColumn::make(__('Image'), 'image')
  ->location(
      fn ($row) => dd($row->image) //asset('storage/img/menu/' . $menu->image)
    ),
```

And why if I don't use type hint I have to name the variable with $row? if with other name it will show error "Unable to resolve dependency [Parameter #0 [ <required> $menu ]] in class App\Livewire\MenuTable"

**Comment by lrljoe** [comment]
Add image in setAdditionalSelects() in your configure method for now, as it this will get you working until I can patch this
